lib/Manifestation.inc

```diff
@@ -201,6 +209,28 @@ class Manifestation extends ManifestationMaster {
     return (is_array($result)) ? reset($result) : $result;
   }
 
+  /**
+   * typeGroup
+   * zero or one occurence
+   * @return string|null
+   */
+  public function getTypeGroup() {
+    $result = TingOpenformatMethods::parseFields($this->manifestation->adminData, array('typeGroup'));
+
+    return (is_array($result)) ? reset($result) : $result;
+  }
+
+  /**
+   * holdingsCount
+   * zero or one occurence
+   * @return string|null
+   */
+  public function getHoldingsCount() {
+    $result = TingOpenformatMethods::parseFields($this->manifestation->adminData, array('holdingsCount'));
+
+    return (is_array($result)) ? reset($result) : $result;
+  }
```

Comment:
Slå de her to metoder sammen og giv metoden et input parameter.
Der er otte tegn til forskel spredt ud over 14 linjer kode.

Reply:
tja, men så burde det nok gøres for hele klassen - det er den måde, som det er lavet for rigtig mange elementer...